ingeniamc/k2-base-camp
# DR0053A-571 Update docs

### Docs

https://novantamotion.atlassian.net/browse/DR0053A-571

The documentation and README of the project should accurately reflect the current state. In particular, images of the interface should accurately represent what the user sees and any commands that are mentioned should work as intended.

### Test

The user should be able to follow the README and/or documentation without prior knowledge and be able to:

- Run the program
- Generate the style file
- Install and successfully run pre-commit hooks
- Compile and view the documentation
- Successfully run tests

Compiling the documentation:

1. `cd docs`
2. `make html`
3. the "build/html"-folder should have an "index.html" that can be opened in your browser to view the documentation
